common/smoke-test/Initialize-SmokeTests.ps1

```diff
@@ -90,20 +90,28 @@ function Set-EnvironmentVariables {
 
 function New-DeployManifest {
   Write-Verbose "Detecting samples..."
-  $javascriptSamples = (Get-ChildItem -Path "$repoRoot/sdk/$ServiceDirectory/*/samples/javascript/" -Directory
-    | Where-Object { Test-Path "$_/package.json" })
+  $packageDir = Get-ChildItem -Directory -Path "$repoRoot/sdk/$ServiceDirectory/*"
+
+  $javascriptSamples = $packageDir | ForEach-Object {
```

Comment:
While in scripts I generally prefer foreach loop over the ForEach-Object pipeline operator. Thanks are easier to understand.  Also based on experience we have mostly moved away from using pipelines (i.e. `|`) in our script files because we have learn they have some unintentional side-effects when it comes to error handling and other aspects.